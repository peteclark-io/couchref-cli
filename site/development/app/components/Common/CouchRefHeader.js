'use strict'

import React from 'react';
import bootstrap from 'bootstrap/dist/css/bootstrap.css';
import {Link} from 'react-router';

import styles from './styles.css';

const CouchRefHeader = React.createClass({

  render: function() {
    return (
      <div className={styles['brand-header']}>
        <div className={bootstrap.container}>
          <div className={bootstrap.row}>
            <div className={bootstrap['col-xs-4']}>
              <Link to={`/`}><h1>CouchRef</h1></Link>
            </div>
            <div className={bootstrap['col-xs-8']}>
            </div>
          </div>
        </div>
      </div>
    );
  }
});

export default CouchRefHeader;
