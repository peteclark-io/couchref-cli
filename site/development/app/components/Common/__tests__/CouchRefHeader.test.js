'use strict';

import React from 'react';
import CouchRefHeader from '../CouchRefHeader';
import renderer from 'react-test-renderer';

it('Renders ok.', () => {
   const rendered = renderer.create(
      <CouchRefHeader />
   );
   expect(rendered.toJSON()).toMatchSnapshot();
});
