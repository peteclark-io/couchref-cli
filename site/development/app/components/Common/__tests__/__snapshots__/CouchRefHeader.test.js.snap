exports[`test Renders ok. 1`] = `
<div
  className="brand-header">
  <div
    className="container">
    <div
      className="row">
      <div
        className="col-xs-4">
        <a
          onClick={[Function]}
          style={Object {}}>
          <h1>
            CouchRef
          </h1>
        </a>
      </div>
      <div
        className="col-xs-8">
        <a
          onClick={[Function]}
          style={Object {}}>
          <h3
            className="pull-right">
            Profile
          </h3>
        </a>
        <a
          onClick={[Function]}
          style={Object {}}>
          <h3
            className="pull-right">
            Your Score
          </h3>
        </a>
      </div>
    </div>
  </div>
</div>
`;
